### PR TITLE
Oleksii/numbers

### PR DESCRIPTION
**Pull request by alexone55**

### CODE REVIEW

**Comment by glock3** [comment]
Hi @alexone55 
All file names should be in snake case 
https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

File: Alexey/Calculator.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
+def main():
+    answer = input("Choose operation d/m/e/a/r: ")
```
Comment (glock3) [inline Alexey/Calculator.py]: It doesn't work

File: Alexey/Change Return Program.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+
+def main():
+    total = float(input("How much money you need to change: ")) * 100
```
Comment (glock3) [inline Alexey/Change Return Program.py]: I doesn't work 
Test:
How much money you need to change: 7
Quarters: 28 
Dimes: 0 
Nickels: 0
Pennies: 0

File: Alexey/Fibonachi.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+def main():
+    digit = int(input("Please type count of digits: "))
```
Comment (glock3) [inline Alexey/Fibonachi.py]: It doesn't work
Task: Fibonacci Sequence - Enter a number and have the program generate the Fibonacci sequence to that number or to the Nth number.
Test:
Please type count of digits: 5
1
1
2
3
5
8
13
21
34
55
89
144
233
377
610
987
1597
2584
4181
6765
10946
17711
28657
46368
75025
Expected:
5

Test:
Please type count of digits: 2
1
1
2
3
5
8
13
21
34
55
89

Expected: 1

File: Alexey/Mortgage Calculator.py [hunk 4]
```diff
@@ -0,0 +1,42 @@
+answer = input("Choose type of calc y/m/w/d/: ")
+P = int(input("Principal Amount (initial loan balance): "))
+i = float(input("Interest Rate: ")) / 100
+t = int(input("Count of Years: "))
+n = t * 12
```
Comment (glock3) [inline Alexey/Mortgage Calculator.py]: This should be moved to main method or to the `if __name__ == "__main__":` statement

File: Alexey/Number_e.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+def main():
+    digit = int(input("Please type count of digits: "))
+    number_e(digit)
+
+def number_e(digit):
+    e = 0
+
+    for i in range(1, 100000000):
+        e += ((1+(1/ i)) ** i)
+        e = e / i
```
Comment (glock3) [inline Alexey/Number_e.py]: Is it possible to speedup this calculations or to simplify a bit?

File: Alexey/Number_pi.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    number_pi(digit)
+
+
+def number_pi(digit):
+
+    pi = 1
+
+    for i in range(1, 100000000):
+        pi += ((-1) ** i) * (1 / (2 * i +1))
+
```
Comment (glock3) [inline Alexey/Number_pi.py]: Is it possible to speedup this calculations or to simplify a bit?

File: Alexey/Tax Calculator.py [hunk 7]
```diff
@@ -0,0 +1,14 @@
+def main():
```
Comment (glock3) [inline Alexey/Tax Calculator.py:1]: It doesn't work correctly 
Test:
Type number:120
Type number:50
180.0
Expected:
240

File: Alexey/Change Return Program.py (hunk 2 above)
Comment (alexone55) [reply Alexey/Change Return Program.py]: Sorry, I have not specified input in dollars, 7 dollars its 700 cents and 28 of Quarters

**Pull request closed without merging**